components/patient-location-stats.tsx

```
@@ -0,0 +1,394 @@
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+  TooltipProps,
+} from "recharts"
+import {
+  MapPin,
+  TrendingUp,
+  Users,
+  Globe,
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)